quark/subproject.py

```diff
@@ -75,6 +75,8 @@ def _parse_fragment(url):
         res = {}
         for equality in url.fragment.split("&"):
             index = equality.find('=')
+            if index < 0:
+                continue
```

Comment:
Shouldn't it be something like
```
    key = equality
    value = ''
else
```
...
instead?

Reply:
Last version should have the same effect as this proposed change.